page-title.html

```diff
@@ -80,12 +80,20 @@
     ],
 
     _updatePageTitle: function(baseTitle, divider, pageTitle, direction) {
-      var pieces;
+      var pieces = [];
+
+      if (pageTitle) {
+        pieces.push(pageTitle);
+      }
 
       if (direction == 'standard') {
-        pieces = (baseTitle) ? [baseTitle, pageTitle] : [pageTitle];
+        if (baseTitle) {
+          pieces.unshift(baseTitle)
+        }
       } else if (direction == 'reversed') {
-        pieces = (baseTitle) ? [pageTitle, baseTitle] : [pageTitle];
+        if(baseTitle) {
```

Comment:
Could we get a space between `if` and `(` so it matches the style of the rest of the code? Probably just a typo.